modules/test/playwright/tests/object-web/objectEntries.spec.ts

```diff
@@ -111,6 +111,89 @@ test.describe('Manage object entries through page templates', () => {
 		);
 	});
 
+	test('can view all entries related to an object in the relationship field using autocomplete', async ({
```

Comment:
Hey @p-albuquerque, can you move this test to inside of `test.describe` => **Manage object entries through View Object Entries**